.stylelintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": [
+    "stylelint-config-standard",
+    "stylelint-config-prettier"
+  ],
+  "rules": {
+    "selector-pseudo-element-colon-notation": "double"
+  }
+}
```

Comment:
![image](https://user-images.githubusercontent.com/58953933/112004046-ad4fe800-8ade-11eb-9799-d5e0f2ae8f1c.png)

Reply:
Let's improve Stylelint/Prettier configuration in a separate task.
https://mzhukov.atlassian.net/browse/LID-5